### PR TITLE
new data source quota rules

### LINKED ISSUE #135: [New Data Source]: /storage/quota/rules
### Description

Add a data source for /storage/quota/rules

### Requested Resource(s) and/or Data Source(s)

netapp-ontap_storage_quota_rules

### Potential Terraform Configuration

_No response_

### References

https://docs.netapp.com/us-en/ontap-restapi/ontap/swagger-ui/index.html#/storage/quota_rule_collection_get

### Would you like to implement a fix?

None

### PR DESCRIPTION
**Pull request by suhasbshekar**
#135

### CODE REVIEW

File: docs/data-sources/storage_quota_rule_data_source.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "netapp-ontap_quota_rule Data Source - terraform-provider-netapp-ontap"
+subcategory: "Storage"
+description: |-
+  StorageQuotaRule data source
+---
+
+# netapp-ontap_quota_rule (Data Source)
+
+StorageQuotaRule data source
```
Comment (carchi8py) [inline docs/data-sources/storage_quota_rule_data_source.md]: change this name no storage (same thing on lines 6)

File: docs/data-sources/storage_quota_rules_data_source.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "netapp-ontap_quota_rules Data Source - terraform-provider-netapp-ontap"
+subcategory: "Storage"
+description: |-
+  StorageQuotaRules data source
```
Comment (carchi8py) [inline docs/data-sources/storage_quota_rules_data_source.md]: change this name (no storage), same on line 11

**Review by carchi8py — CHANGES_REQUESTED** [review]

**Review by carchi8py — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,8 @@
 FEATURES:
 * **New Data Source:** `netapp-ontap_volumes_files` ([#8](https://github.com/NetApp/terraform-provider-netapp-ontap/issues/8))
 * **New Resource:** `netapp-ontap_quota_rules` ([#136](https://github.com/NetApp/terraform-provider-netapp-ontap/issues/136))
+* **New Data Source:** `netapp-ontap_quota_rules` ([#135](https://github.com/NetApp/terraform-provider-netapp-ontap/issues/135))
```
Comment (chuyich) [inline CHANGELOG.md:5]: Just a nitpick. Based on what we have, the data source and resource are grouped together. 
So this line should be moved up to line 5.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,8 @@
...
+* **New Data Source:** `netapp-ontap_quota_rule` ([#135](https://github.com/NetApp/terraform-provider-netapp-ontap/issues/135))
```
Comment (chuyich) [inline CHANGELOG.md:6]: Same as this line. Data source should be grouped together.

**Pull request merged**